applications/matter_bridge/src/app_task.cpp

```diff
@@ -149,13 +151,13 @@ CHIP_ERROR AppTask::Init()
 	ConfigurationMgr().LogDeviceConfig();
 	PrintOnboardingCodes(chip::RendezvousInformationFlags(chip::RendezvousInformationFlag::kBLE));
 
-	/* Initialize bridge manager */
-	BridgeManager::Instance().Init();
-
 #ifdef CONFIG_BRIDGED_DEVICE_BT
 	BLEConnectivityManager::Instance().Init(sUuidServices, kUuidServicesNumber);
 #endif
 
+	/* Initialize bridge manager */
```

Comment:
Yes, it is because the `BridgeManager` calls `RestoreBridgedDevices` on init and after loading Bluetooth LE devices from the storage it tries to reconnect to them, so `BLEConnectivityManager` must be ready for it.

Reply:
Can we at least document that as a comment?

Reply:
Sure